src/appleseed/foundation/utility/api/apistring.cpp

```diff
@@ -42,7 +43,7 @@ namespace foundation
 // APIString class implementation.
 //
 
-APIString::APIString()
+APIString::APIString() APPLESEED_NOEXCEPT
```

Comment:
Are `noexcept` specifiers necessary on method _definitions_? Can't they be limited to method _declarations_?

Reply:
They are necessary.